giskard/scanner/robustness/text_transformations.py

```diff
@@ -122,6 +122,57 @@ def _random_key_typo(self, char):
             return typo if char.islower() else typo.upper()
         return char
 
+class TextFromSpeechTypoTransformation(TextTransformation):
+    name = "Add text from speech typos"
+    
+    def __init__(self, column, rate=0.05, min_length=10, rng_seed=1729):
+        super().__init__(column)
+        from .entity_swap import speech_typos
+
+        self.rate = rate
```

Comment:
It seems that we are not using this rate and instead make typo to all words matched in the dict

Reply:
I have made the changes as requested. Do let me know if you would like me to add or change anything else.

Reply:
Thanks, it's good!